docs/coordinates/solarsystem.rst

```diff
@@ -44,6 +44,52 @@ without the need to download a large ephemerides file)::
 Above, we used ``solar_system_ephemeris`` as a context, which sets the default
 ephemeris while in the ``with`` clause, and resets it at the end.
 
+.. note::
+  The algorithm is due to J.L. Simon, P. Bretagnon, J. Chapront,
```

Comment:
I'd say "The algorithm used by ERFA is ..."

Also, a nitpick, but could you use consistent line breaks, so that the text remains easy to read also in editors?